.umirc.ts

```diff
@@ -7,5 +7,6 @@ export default defineConfig({
     'https://avatars0.githubusercontent.com/u/9441414?s=200&v=4',
   logo:
     'https://avatars0.githubusercontent.com/u/9441414?s=200&v=4',
-  outputPath: '.doc'
+  outputPath: '.doc',
+  ssr: {}
```

Comment:
开启 `exportStatic` 会按照路由输出 HTML

```suggestion
  ssr: {},
  exportStatic: {}
```

另外 `now.json` 应该要配置下 routes：https://github.com/umijs/dumi/blob/master/now.json#L12

Reply:
好的，我试试